frontend/src/app/mymeetups/own-meetups/own-meetups.component.html

```diff
@@ -0,0 +1,18 @@
+<h2>Meine Meetups</h2>
```

Comment:
Ich habe jeweils das 'component' im Filenamen der html Files weggelassen. Für mich ist es so angenehmer zu lesen in der Nav.

Reply:
Neuer Name own-meetups.html